OVERLOAD.md

```diff
@@ -2,7 +2,6 @@
 
 ## Add Deepl automatic translation
```

Comment:
@Quentinchampenois I deliberately deleted the overrides for machine translation because it seems to me that the 0.24 as the same files. @MoretS Could you double check those deletions before delivery with the DB?
Especially the versioning problem we had with machine translations (same than people powered)

Reply:
I also changed the footer layout overrides to make them match the ones from the decidim-app master branch.